frontend/packages/db-structure/src/parser/sql/postgresql/index.test.ts

```diff
@@ -171,15 +171,90 @@ describe(processor, () => {
       expect(value).toEqual(parserTestCases['index (unique: true)']('btree'))
     })
 
-    // FIXME: `CONSTRAINT` statement is not supported yet
-    it.skip('foreign key constraint in CREATE TABLE', async () => {
-      await processor(/* sql */ `
+    it('table-level primary key constraint with named constraint', async () => {
+      const { value } = await processor(/* sql */ `
+        CREATE TABLE bar (
+          bar_id int,
+          name text NOT NULL,
+          CONSTRAINT bar_primary_key PRIMARY KEY (bar_id)
+        );
+      `)
+
+      expect(value.tables['bar']?.constraints).toEqual({
+        bar_primary_key: {
+          name: 'bar_primary_key',
+          type: 'PRIMARY KEY',
+          columnName: 'bar_id',
+        },
+      })
+    })
+
+    it('table-level primary key constraint without named constraint', async () => {
+      const { value } = await processor(/* sql */ `
+        CREATE TABLE foo_bar (
+          foo_id int,
+          bar_id int,
+          PRIMARY KEY (foo_id, bar_id)
+        );
+      `)
+
+      expect(value.tables['foo_bar']?.constraints).toEqual({
+        PRIMARY_foo_id: {
+          name: 'PRIMARY_foo_id',
+          type: 'PRIMARY KEY',
+          columnName: 'foo_id',
+        },
+        PRIMARY_bar_id: {
+          name: 'PRIMARY_bar_id',
+          type: 'PRIMARY KEY',
+          columnName: 'bar_id',
+        },
+      })
+    })
+
+    it('table-level single column primary key constraint', async () => {
+      const { value } = await processor(/* sql */ `
+        CREATE TABLE baz (
+          baz_id int,
+          name text,
+          PRIMARY KEY (baz_id)
+        );
+      `)
+
+      expect(value.tables['baz']?.constraints).toEqual({
+        PRIMARY_baz_id: {
+          name: 'PRIMARY_baz_id',
```

Comment:
It was found that the default naming when constraint name is omitted is different from PostgreSQL's behavior. However, this is a problem that also occurs in existing implementations, so it is not fixed here and will be addressed as a separate issue.

ref: https://github.com/liam-hq/liam/pull/2258#issuecomment-3011259021